### PR TITLE
Adding `spacy-cleaner` to the spaCy universe

### PR DESCRIPTION
**Pull request by Ce11an**
## Description

I have built a package that extends spaCy for text processing called [spacy-cleaner](https://github.com/Ce11an/spacy-cleaner). I would like to add it to the spaCy universe!

I have tried to build the website locally with npm/docker. I have an M1 MacBook Pro and read the docs, but I am having no luck running anything successfully. Can someone advise on what I should do?

### Types of change

📚 Change to documentation (website).

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all the required information.

### CODE REVIEW

File: website/meta/universe.json [hunk 1]
```diff
@@ -4072,6 +4072,47 @@
...
+            "code_example": [
+                "import spacy",
+                "import spacy_cleaner",
+                "from spacy_cleaner.processing import removers, replacers, mutators",
+                "",
+                "model = spacy.load(\"en_core_web_sm\")",
+                "pipeline = spacy_cleaner.Pipeline(",
+                "    model,",
+                "    removers.remove_stopword_token,",
+                "    replace.replace_punctuation_token,",
```
Comment (polm) [inline website/meta/universe.json:4092]: Looks like a typo?

```suggestion
                "    replacers.replace_punctuation_token,",
```

**Comment by polm** [comment]
Thanks for this PR, this looks like a neat project - we often have users ask about these tasks. 

There were some issues of the PR so I made some modifications.

- it looks like `master` was a bit old, so a merge was needed
- info was in the wrong part of `universe.json` (this is our fault for not making this clearer)
- there seems to have been a typo
- minor other stuff

About running the site, it is a bit finicky to get working, but what works for me is using node `v10.15.0` with nvm. If you were doing that and it wasn't working it may have been because your info was in the wrong section of `universe.json`, which would break the build for the universe part.

Anyway, let me know if the changes look OK and this should be good to add.

**Comment by Ce11an** [comment]
Hey @polm,

Thanks for the assistance - much appreciated!

I still cannot get the website to build locally, but if you are happy to get this merged, then so am I 😄

**Comment by polm** [comment]
Got it, I'll go ahead and merge this then.

One comment about your project though - while there are still good reasons to do things like remove stop words and lemmatize words, sometimes we have users do that and then wonder why our pretrained NER / tagger etc. don't work on the processed text. If you could add a comment or two in your docs about potential issues with that that'd be appreciated. (We covered this a little in an FAQ [here](https://github.com/explosion/spaCy/discussions/10243))

**Pull request merged**